section3/exercises/arrays.rb

```diff
@@ -25,43 +25,51 @@
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
-
+animals.each do |animals|
+  puts "Type of animal : #{animals}"
+end
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
-
+animals[2] = "Gorilla"
 
 # YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
+animals[3] = "Lion"
 
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
+print animals[-1]
 
 #-------------------
 # PART 2: Foods: Array Methods
 #-------------------
 
 # YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+food = ["Banana", "Orange", "Apple", "Peach"]
 
 # YOU DO: Write code below that will print the number of elements in the array of
 # foods from above.
+food.each do |food|
+  puts "Examples of different types of food : #{food}"
+end
 
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
-
+food[4] = "Broccoli"
 
 # YOU DO: Write code below that removes the last item of food from the foods array and
 # print the changed array to verify that item has been removed
+food.delete_at(-1)
 
-
-# YOU DO: Write code to add 3 new foods to the array. 
+# YOU DO: Write code to add 3 new foods to the array.
   # There are several ways to do this - choose whichever you'd like!
 # Then, print the changed array to verify the new items have been added
-
+food << "Apricot"
+food << "Watermelon"
+food << "Dragonfruit"
+print food
 # YOU DO: Remove the food that is in index position 0.
-
+food.delete_at(0)
```

Comment:
There's another method that we might see more often in this use case. See if you can find it on the [ruby-doc.org Array class page](https://ruby-doc.org/core-2.7.2/Array.html).